aiven/client/client.py

```diff
@@ -602,7 +605,9 @@ def update_service_topic(
                 "replication": replication,
                 "retention_bytes": retention_bytes,
                 "retention_hours": retention_hours,
-            },
+            } | ({
```

Comment:
This doesn't seem right, dictionaries don't support he or operator:

```python
>>> d = {1:1}
>>> d2 = {2:2}
>>> d | d2
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unsupported operand type(s) for |: 'dict' and 'dict'
```
You probably want to use `dict.update`.

Reply:
I didn't notice that is Python 3.9 new feature...

Reply:
And I didn't know that 3.9 introduced it :smile:  https://www.python.org/dev/peps/pep-0584/

But 3.6, 3.7, and 3.8 are supported versions:

https://github.com/aiven/aiven-client/blob/0ec1a2b02c0863d2fbfc42298d437389b1e87aac/setup.py#L50-L52

So we shouldn't use it here